src/source/location.d

```diff
@@ -94,11 +94,11 @@ package:
 		return !_mixin;
 	}
```

Comment:
It does look like this should be public as well, then.

Reply:
done